engine/frontend/src/utils/download.js

```diff
@@ -0,0 +1,27 @@
+export const saveTextAsFile = (text, fileName) => {
+    const blob = new Blob([text], {type: "text/plain"});
+    const downloadLink = document.createElement("a");
```

Comment:
can we make `"a"` a default parameter so it can be overwritten. This would be to avoid collisions with other elements :)